msicreator/create_installer.py

```diff
@@ -13,11 +13,15 @@ def prepare_project_dir():
     new_fonts_dir = os.path.join(creator_dir, 'libo-fonts/share/fonts')
     main_dir = os.path.join(creator_dir, 'main')
     src_uninstaller_icon = os.path.join(src_dir, 'icon-themes/colibre/res/mainapp_48_8.png')
+    src_ui_banner = os.path.join(src_dir, 'instsetoo_native/inc_common/windows/msi_templates/Binary/Banner.bmp')
```

Comment:
Integrated the UI graphics feature into create_installer.py. 
It was tested and worked fine, please test it again @Rakielle